nearcore/Cargo.toml

```diff
@@ -72,20 +68,10 @@ protocol_feature_add_account_versions = ["near-primitives/protocol_feature_add_a
 protocol_feature_tx_size_limit = ["near-primitives/protocol_feature_tx_size_limit", "node-runtime/protocol_feature_tx_size_limit"]
 protocol_feature_allow_create_account_on_delete = ["node-runtime/protocol_feature_allow_create_account_on_delete"]
 protocol_feature_fix_storage_usage = ["near-primitives/protocol_feature_fix_storage_usage", "node-runtime/protocol_feature_fix_storage_usage"]
-nightly_protocol_features = ["nightly_protocol", "near-primitives/nightly_protocol_features", "near-client/nightly_protocol_features", "near-epoch-manager/nightly_protocol_features", "near-store/nightly_protocol_features", "protocol_feature_evm", "protocol_feature_block_header_v3", "protocol_feature_alt_bn128", "protocol_feature_add_account_versions", "protocol_feature_tx_size_limit", "protocol_feature_allow_create_account_on_delete", "protocol_feature_fix_storage_usage", "protocol_feature_restore_receipts_after_fix"]
+nightly_protocol_features = ["nightly_protocol", "near-primitives/nightly_protocol_features", "near-client/nightly_protocol_features", "near-epoch-manager/nightly_protocol_features", "near-store/nightly_protocol_features", "protocol_feature_evm", "protocol_feature_block_header_v3", "protocol_feature_alt_bn128", "protocol_feature_add_account_versions", "protocol_feature_tx_size_limit", "protocol_feature_allow_create_account_on_delete", "protocol_feature_fix_storage_usage", "protocol_feature_restore_receipts_after_fix", "protocol_feature_cap_max_gas_price"]
 nightly_protocol = ["near-primitives/nightly_protocol", "near-jsonrpc/nightly_protocol"]
-protocol_feature_restore_receipts_after_fix = ["near-primitives/protocol_feature_restore_receipts_after_fix", "near-chain/protocol_feature_restore_receipts_after_fix", "node-runtime/protocol_feature_restore_receipts_after_fix", "restored-receipts-verifier/protocol_feature_restore_receipts_after_fix"]
-
-# enable this to build neard with wasmer 1.0 runner
-# now if none of wasmer0_default, wasmer1_default or wasmtime_default is enabled, wasmer0 would be default
-wasmer1_default = ["node-runtime/wasmer1_default"]
-wasmer0_default = ["node-runtime/wasmer0_default"]
-wasmtime_default = ["node-runtime/wasmtime_default"]
-# enable some of these to run runtime tests with wasmer 1.0, 0.x and wasmtime vm enabled
-# but would not change default runner used by neard
-wasmer1_vm = ["node-runtime/wasmer1_vm"]
-wasmer0_vm = ["node-runtime/wasmer0_vm"]
-wasmtime_vm = ["node-runtime/wasmtime_vm"]
```

Comment:
Aren't these only used for the tests in `integration-tests`? I moved them there.

Reply:
@matklad @olonho are these documented knobs that we want someone to play around easily?

Reply:
Well, think as with most of our features, they are ad-hoc knobs to configure certain behavior.

Reply:
Yeah, we should leave this features here so that we can decide to build `neard` with `wasmer0` or `wasmer1`. 

On the other hand, I don't think we need this set of features in `integration-tests/Cargo.toml` -- integration tests should use whatever our default VM is. We test several different vms inside `near_vm_runner`.